tests/all_coverage.json

```diff
@@ -0,0 +1,3 @@
+{
+   "include": ["pcmdi_metrics", "cdat_info", "cdms2", "genutil", "vcs", "vtk"]
```

Comment:
we do not want coverage on anything but pcmdi_metrics for this repo.

Reply:
actually I'm not sure what this is for?

Reply:
when we run 'pcmdi_metrics' testsuite with coverage, it will only collect coverage for 'pcmdi_metrics'. (specified in tests/coverage.json).  
In cdat nightly, where we run all test suites, I want to collect coverage for all packages...so that we can see our overall coverage...for ex: pcmdi_metrics test cases uses cdms2, genutil, vcs, etc... coverage from running all test suites will be collected....and for that I will use tests/all_coverage.json.